ui/app/AppLayouts/Chat/views/ChatHeaderContentView.qml

```diff
@@ -259,7 +260,7 @@ RowLayout {
                 case Constants.chatType.publicChat:
                     return qsTr("Public chat")
                 case Constants.chatType.privateGroupChat:
-                    let cnt = root.usersStore.usersModule.model.count
+                    let cnt = chatContentModule.usersModule.model.count
                     if(cnt > 1) return qsTr("%n member(s)", "", cnt);
                     return qsTr("1 member");
```

Comment:
Existing code I know but this whole block is wrong, it should be:
```qml
return qsTr("%n member(s)", "", chatContentModule.usersModule.model.count)
```
There should never be a special case for any number

Reply:
If we can't have less than two members in a group then drop the parentheses in "member((s)"

Reply:
Nope, they must stay, always :) 

https://doc.qt.io/qt-6/i18n-source-translation.html#handling-plurals

Reply:
Thanks for the heads-up, forgot about it